web/src/lib/utils/byte-units.ts

```diff
@@ -12,5 +22,24 @@ export function getHumanReadableBytes(bytes: number): string {
 		}
 	}
 
-	return `${remainder.toFixed(magnitude == 0 ? 0 : 1)} ${units[magnitude]}`;
+	remainder = parseFloat(remainder.toFixed(maxPrecision));
+
+	return [remainder, units[magnitude]];
+}
+
+/**
+ * Localized number of bytes with a unit.
+ *
+ * For `1536` bytes:
+ * * en: `1.5 KiB`
+ * * de: `1,5 KiB`
+ *
+ * @param bytes number of bytes
+ * @param maxPrecision maximum number of decimal places, default is `1`
+ * @returns localized bytes with unit as string
+ */
+export function asByteUnitString(bytes: number, maxPrecision = 1): string {
+	const locale = Array.from(navigator.languages);
+	const bytesWithUnit = getBytesWithUnit(bytes, maxPrecision);
```

Comment:
Can you use array deconstruction here? Or, change the return to be an object and then use object deconstruction?